scripts/sync-results.py

```diff
@@ -27,10 +27,9 @@ def main() -> None:
         description="Sync data from an analysis module to your S3 bucket.",
     )
     parser.add_argument(
-        "--module",
-        "-m",
+        "module",
+        metavar="MODULE",
```

Comment:
noticed while I was here - maybe change the help below to "The name of the analysis module to sync **RESULTS** from.", (not data)

Reply:
hmmm... "to sync results _for_"? either way.